packages/web/components/swap-tool/index.tsx

```diff
@@ -353,13 +426,32 @@ export const SwapTool: FunctionComponent<{
     return (
       <>
         {ads && featureFlags.swapsAdBanner && <AdBanner ads={ads} />}
-        <div className="relative flex flex-col gap-8 overflow-hidden rounded-3xl bg-osmoverse-800 px-6 py-8 md:gap-6 md:px-3 md:pt-4 md:pb-4">
+        <div className="relative flex flex-col gap-6 overflow-hidden rounded-3xl bg-osmoverse-850 px-6 py-9 md:gap-6 md:px-3 md:pt-4 md:pb-4">
           <Popover>
             {({ open, close }) => (
               <>
                 <Popover.Overlay className="absolute inset-0 z-40 !rounded-3xl bg-osmoverse-1000/80" />
                 <div className="relative flex w-full items-center justify-end">
-                  <h6 className="w-full text-center">{t("swap.title")}</h6>
+                  <div className="flex w-full px-2">
+                    <SwapToolHeaderClickableItem
+                      title={t("swap.title")}
+                      ariaLabel="Change to swap tab"
+                      active={isSwapTab}
+                      onClick={() => {
+                        setIsSwapTab(true);
+                      }}
+                    />
+                    <SwapToolHeaderClickableItem
+                      title={t("swap.buy")}
+                      ariaLabel="Open buy modal"
+                      active={!isSwapTab}
+                      onClick={() => {
```

Comment:
We have to enable it only if the wallet is connected, because if you try to click and select "kado," it will not start, but if you have the wallet connected it will work.